qa/tasks/cephfs/test_volume_client.py

```diff
@@ -905,3 +905,93 @@ def test_recover_metadata(self):
             volume_id=volume_id,
             auth_id=guestclient["auth_id"],
         )))
+
+    def test_put_object(self):
+        vc_mount = self.mounts[1]
+        vc_mount.umount_wait()
+        self._configure_vc_auth(vc_mount, "manila")
+
+        obj_data = b'test data'
+        obj_name = 'test_obj'
+        pool_name = self.fs.get_data_pool_names()[0]
+        self._volume_client_python(vc_mount, dedent("""
+            vc.put_object("{pool_name}", "{obj_name}", b"{obj_data}")
+        """.format(
+            pool_name = pool_name,
+            obj_name = obj_name,
+            obj_data = obj_data
+        )))
+
+        temp_bin_path = '/tmp/' + obj_name + '.bin'
+        self.fs.client_remote.run(args=[
+            'sudo', os.path.join(self.fs._prefix, 'rados'), '-p', pool_name,
+            'get', obj_name, temp_bin_path
+        ])
+        with open(temp_bin_path, 'rb') as f:
+            read_data = f.read()
+        self.assertEqual(obj_data, read_data)
+
+    def test_get_object(self):
+        vc_mount = self.mounts[1]
+        vc_mount.umount_wait()
+        self._configure_vc_auth(vc_mount, "manila")
+
+        obj_data = b'a'
+        obj_name = 'test_obj_1'
+        pool_name = self.fs.get_data_pool_names()[0]
+        temp_bin_path = '/tmp/' + obj_name + '.bin'
+        with open(temp_bin_path, 'wb') as f:
```

Comment:
The reason the test failed is because the file is created by the machine executing the test and not by the machine acting as the client.

See `qa/tasks/cephfs/mount.py` for examples.

Reply:
Thanks, Patrick. Fixed this.